### ISSUE TITLE
Bug in LOC facet

### ISSUE THREAD

**Issue by jacobthill** [sul-dlss/dlme-transform#478]
Filtering by 'Library of Congress' from the 'Source Institution' or 'Holding Institution' facet results in the following error message: 

<img width="1030" alt="Screen Shot 2020-02-13 at 12 58 38 PM" src="https://user-images.githubusercontent.com/37662787/74464054-da07e980-4e60-11ea-9cce-553b00dca2fa.png">

**Comment by cbeer** [comment]
I assume it's related  to https://app.honeybadger.io/projects/53082/faults/60777823; it looks like  the thumbnail for at least one record in that collection was mapped as `http:https://www.loc.gov/static/collections/abdul-hamid-ii/155_155.jpg`

**Comment by jacobthill** [comment]
Thanks @cbeer.

**Comment by aaron-collier** [comment]
Should this be moved to dlme or is it really a dlme-transform issue?

**Comment by jacobthill** [comment]
@cbeer how can I create track down these errors myself? I fixed the error you noted, I think, but I'm getting the same behavior. I created a free honeybadger account but the url you linked redirects to this: https://app.honeybadger.io/first_run/install_code.

**Comment by jacobthill** [comment]
@aaron-collier its probably a mapping problem though we might want different behavior when there is a mapping error. I'll defer to you and @cbeer.

**Comment by jacobthill** [comment]
I think this ticket is a symptom of a larger problem: I often can't get the information I need to identify the source of problems so I can fix them. We may need to open another ticket to brief me and get me the necessary credentials to view the application logs. I think this would save everyone time in the long run.

**Comment by jacobthill** [comment]
I think I figured out one major problem. When loading Harvard data I got 12,622 records, then I changes the url for the agg_preview (https://github.com/sul-dlss/dlme-transform/commit/a2e67b5a6c05eca0bd2280613b88e1390c3992ef) and only got 747. It rejected the whole collection that the change was applied to. It gets a bad url and won't load the record. This is the same problem with LOC above and probably the other collections as well, though possibly not all of them. Here is the honeybadger log: https://app.honeybadger.io/projects/53082/faults/61172349. Now that I can see the honeybadger errors I can probably fix this myself, though I will leave this open for now because we may want to discuss the desired behavior upon encountering an unresolvable url.
